buildsystem/HandlePythonOptions.cmake

```diff
@@ -29,7 +29,7 @@ if(NOT CMAKE_PY_INSTALL_PREFIX)
 	if(MSVC)
 		set(CMAKE_PY_INSTALL_PREFIX "python")
 	else()
-		py_exec("import site, os; print(os.path.normpath(site.getsitepackages()[0]))" PREFIX)
+		py_exec("import os; print(os.path.dirname(os.__file__) + '/site-packages')" PREFIX)
```

Comment:
This breaks the installation for distribution packaging because there the destination path must be the sitepackages directory. What is the intention of this fix, or rather: Why doesn't it work with a virtualenv currently? To solve this correctly, we'll need a more complete solution.

Reply:
Please excuse me, you are right, i did not realize the `os.__file__`'s effect.

Maybe we can improve it further to not depend on this "os.py is one folder above site packages":
https://stackoverflow.com/a/122340/1348535
so we would have:
```python
from distutils.sysconfig import get_python_lib; print(get_python_lib())
```